File: reddio-java/reddio-api/src/main/java/com/reddio/api/v1/rest/OrderInfoResponse.java
package com.reddio.api.v1.rest;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor(staticName = "of")
public class OrderInfoResponse {
    @JsonProperty("fee_rate")
    private String feeRate;
    @JsonProperty("base_token")
    private String baseToken;
    @JsonProperty("fee_token")
    private String feeToken;
    @JsonProperty("lower_limit")
    private Long lowerLimit;
    @JsonProperty("nonce")
    private Long nonce;
    @JsonProperty("contracts")
    private List<Contract> contracts;
    @JsonProperty("vault_ids")
    private List<String> vaultIds;
    @JsonProperty("asset_ids")
    private List<String> assetIds;
}

File: reddio-csharp/reddio.api/V1/Rest/OrderInfoResponse.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Reddio.Api.V1.Rest
{
    public class OrderInfoResponse
    {
        public OrderInfoResponse(List<Contract> contracts, List<string> vaultIDs, List<string> assetIDs, string feeRate,
            string baseToken, string feeToken, long lowerLimit, long nonce)
        {
            Contracts = contracts;
            VaultIDs = vaultIDs;
            AssetIDs = assetIDs;
            FeeRate = feeRate;
            BaseToken = baseToken;
            FeeToken = feeToken;
            LowerLimit = lowerLimit;
            Nonce = nonce;
        }

        public OrderInfoResponse()
        {
        }

        [JsonProperty("fee_rate")] public string FeeRate { get; set; }
        [JsonProperty("base_token")] public string BaseToken { get; set; }
        [JsonProperty("fee_token")] public string FeeToken { get; set; }
        [JsonProperty("lower_limit")] public long LowerLimit { get; set; }
        [JsonProperty("nonce")] public long Nonce { get; set; }
        [JsonProperty("contracts")] public List<Contract> Contracts;
        [JsonProperty("vault_ids")] public List<string> VaultIDs;
        [JsonProperty("asset_ids")] public List<string> AssetIDs;
    }
}